examples/production/docker-compose-integration.yml

```diff
@@ -120,20 +147,20 @@ services:
 
   concordion:
     container_name: tr_concordion
-    image: knawhuc/textrepo-concordion:${TR_VERSION}
+    image: knawhuc/textrepo-concordion:${DOCKER_TAG}
     ports:
-    - 8080
+      - 8080
     networks:
-    - textrepo_network
+      - textrepo_network
+    volumes:
+      - concordiondata:/concordion
+      - ~/.m2/repository:/root/.m2/repository
```

Comment:
Is dit een 'quickfix' voor lokaal uitproberen die (onbedoeld?) meelift op deze commit, of is dit een wezenlijk onderdeel?

Reply:
o nee, l157 moet er weer uit...

Reply:
acd6a20